EpiserverRedirects/Caching/ICache.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Forte.EpiserverRedirects.Caching
+{
+    public interface ICache : ICacheRemover
+    {
+        bool TryGet<T>(string key, out T resource) where T : class;
```

Comment:
is resource equals redirect (from method Add)? If yes could you adjust the naming here?